src/toxcore/messenger/packet/mod.rs

```diff
@@ -12,7 +12,7 @@ mod msi;
 mod file_control;
 mod typing;
 mod user_status;
-mod file_data;
+pub mod file_data;
```

Comment:
No need to make it public - we have `pub use`.

Reply:
>use crate::toxcore::messenger::packet::file_data::MAX_FILE_DATA_SIZE;

is used.

Reply:
done